dot/core/syncer_benchmark.go

```diff
@@ -21,17 +22,28 @@ func newBenchmarker(logger log.Logger) *benchmarker {
 }
 
 func (b *benchmarker) begin(block uint64) {
-	b.start = time.Now()
-	b.startBlock = block
+	if !b.syncing {
+		b.start = time.Now()
+		b.startBlock = block
+		b.syncing = true
+	}
 }
 
 func (b *benchmarker) end(block uint64) {
+	if !b.syncing {
+		return
+	}
+
 	duration := time.Since(b.start)
 	blocks := block - b.startBlock
+	if blocks == 0 {
+		blocks = 1
+	}
```

Comment:
I don't understand this, it seem if no blocks were produced, than set it to one block produced?  Why?

Reply:
I noticed that when only one block is being synced, it starts and ends at the same number, causing blocks/second to become 0. alternatively I could change the syncer to call end with `endblock+1`, if that is preferred

Reply:
Ok, that makes sense.  No need to change it.